pkg/web/server.go

```diff
@@ -67,6 +67,7 @@ func New() (*Server, error) {
 	}
 
 	serverInst.serveMux.HandleFunc("GET /{$}", serverInst.requireAuth(serverInst.handleGetHome))
+	serverInst.serveMux.HandleFunc("GET /leaderboard", serverInst.handleGetLeaderboard)
```

Comment:
```suggestion
	serverInst.serveMux.HandleFunc("GET /leaderboard", serverInst.requireAuth(serverInst.handleGetLeaderboard))
```
This'll currently allow unauthenticated users to fetch all of our quotes, 10 quotes at a time

Reply:
Very good catch oops